manager/integration/tests/common.py

```diff
@@ -2827,6 +2829,61 @@ def exec_local(cmd):
     return subprocess.check_output(exec_cmd)
 
 
+def parse_nvmf_endpoint(nvmf):
+    return nvmf[7:].split('/')
+
+
+def get_nvmf_ip(nvmf):
+    nvmf_endpoint = parse_nvmf_endpoint(nvmf)
+    return nvmf_endpoint[0].split(':')[0]
+
+
+def get_nvmf_port(nvmf):
+    nvmf_endpoint = parse_nvmf_endpoint(nvmf)
+    return nvmf_endpoint[0].split(':')[1]
+
+
+def get_nvmf_nqn(nvmf):
+    nvmf_endpoint = parse_nvmf_endpoint(nvmf)
+    return nvmf_endpoint[1]
+
+
+def nvmf_login(nvmf):
+    ip = get_nvmf_ip(nvmf)
+    port = get_nvmf_port(nvmf)
+    # NVMe Qualified Name
+    nqn = get_nvmf_nqn(nvmf)
+
+    cmd_connect = f"nvme connect -t tcp -a {ip} -s {port} -n {nqn}"
+    subprocess.check_output(cmd_connect.split())
+    return wait_for_nvme_device()
+
+
+def nvmf_logout(nvmf):
+    nqn = get_nvmf_nqn(nvmf)
+    try:
+        subprocess.check_call(["nvme", "disconnect", "-n", nqn])
```

Comment:
Just wondering do we have a reference or document with more information or explanations about the nvme commands and nvmf endpoint?

Reply:
The commands are documented at https://github.com/longhorn/longhorn-tests/wiki/Connect-to-the-NVMf-frontend-volume, and the document is referenced in the comments of `nvmf_login`. Thank you.